README.md

```diff
@@ -68,6 +68,23 @@ closure_repositories()
 You are not required to install the Closure Tools or PhantomJS. They will be
 fetched automatically.
 
+### Overriding Dependency Versions
+
+To override the version of any dependency, modify your `WORKSPACE` file to pass`omit_<dependency_name>=True` to `closure_repositories()`. Next define your
```

Comment:
1. Please wrap an column 80.
2. Space after 'pass'
3. It might help if we give the user a little more context. Consider adding a paragraph here that sounds something like: “When you call `closure_repositories()` in your `WORKSPACE` file, it causes a few dozen external dependencies to be added to your project, e.g. Guava, Guice, JSR305, etc. You might need to customize this behavior.”